README.md

```diff
@@ -23,83 +21,57 @@ Currently the operator installs the following products:
 
-Download the integreatly-operator project:
+## Clone the integreatly-operator
 ```sh
 mkdir -p $GOPATH/src/github.com/integr8ly
 cd $GOPATH/src/github.com/integr8ly
 git clone https://github.com/integr8ly/integreatly-operator
 cd integreatly-operator
 ```
 
-If the cluster is not already prepared for the integreatly-operator, you will need to do the following:
-```
-make cluster/prepare/project
-make cluster/prepare/crd
-make cluster/prepare/smtp
-```
-
-* 3scale requires AWS S3 bucket credentials for storage. The bucket should have all public access turned off.
+### Prepare your cluster
 
-Currently this secret (`threescale-blobstorage-<installation-name>`) is created with dummy credentials by the [cloud resource operator](https://github.com/integr8ly/cloud-resource-operator), in the namespace the integreatly operator is deployed into. In order for this feature to work, these credentials should be replaced:
-    * _bucketName_: The name of the AWS bucket
-    * _bucketRegion_: The AWS region where the bucket has been created
-    * _credentialKeyID_: The AWS access key
-    * _credentialSecretKey_: The AWS secret key
+If you are working against a fresh cluster it will need to be prepared using the following:
+Include the INSTALLATION_TYPE if you haven't already exported it
 
-You can use this command to replace S3 credentials in 3Scale secret:
-```sh
-oc process -f deploy/s3-secret.yaml -p AWS_ACCESS_KEY_ID=<YOURID> -p AWS_SECRET_ACCESS_KEY=<YOURKEY> -p AWS_BUCKET=<YOURBUCKET> -p AWS_REGION=eu-west-1 -p NAMESPACE=<integreatly-operator-namespace> -p NAME=threescale-blobstorage-<installation-name> | oc replace -f -
 ```
-
-* Backup jobs require AWS S3 bucket credentials for storage. A `backups-s3-credentials` Secret is created the same way as a 3Scale secret described above.
-
-You can use this command to replace S3 credentials in backup secret:
-```sh
-oc process -f deploy/s3-secret.yaml -p AWS_ACCESS_KEY_ID=<YOURID> -p AWS_SECRET_ACCESS_KEY=<YOURKEY> -p AWS_BUCKET=<YOURBUCKET> -p AWS_REGION=eu-west-1 -p NAMESPACE=<integreatly-operator-namespace> | oc replace -f -
+INSTALLATION_TYPE=<managed/managed-api> make cluster/prepare/local
 ```
 
-### RHMI custom resource
-An `RHMI` custom resource can now be created which will kick of the installation of the integreatly products, once the operator is running:
-```sh
-# Create the installation custom resource
-oc create -f deploy/crds/examples/rhmi.cr.yaml
-
-# The operator can now be run locally
-make code/run
-```
-*Note:* if an operator doesn't find RHMI resource, it will create one (Name: `rhmi`).
+### Configuration options
 
-### Logging in to SSO
+USE_CLUSTER_STORAGE - true or false
+IN_PROW - true or false
 
-In the OpenShift UI, in `Projects > redhat-rhmi-rhsso > Networking > Routes`, select the `sso` route to open up the SSO login page.
 
-# Bootstrap the project
+### Run integreatly-operator
+Include the INSTALLATION_TYPE if you haven't already exported it
 
-```sh
-make cluster/prepare/local
```

Comment:
May be my ignorance but how are the crds and  secrets and service accounts copied to the cluster if this is removed

Reply:
It's included higher up. This is just the gitt diff being wierd. It's on line 56 in the new copy.

Reply:
My bad :+1: